conduit/init_provider.go

```diff
@@ -18,6 +18,11 @@ func MakePipelineInitProvider(currentRound *sdk.Round, genesis *sdk.Genesis) *Pi
 	}
 }
 
+// SetGenesis updates the genesis block in the init provider
+func (a *PipelineInitProvider) SetGenesis(genesis *sdk.Genesis) {
```

Comment:
This could probably be private so that nobody is tempted to call it outside of the pipeline package.

Reply:
This can't be private--it's used across a bunch of places.